--- /home/app/config.js
+++ /home/app/config.js
@@ -33,6 +33,8 @@ internals.Config.pg = {
 internals.Config.rabbitMQ = {
   host: process.env.UNMS_RABBITMQ_HOST || '127.0.0.1',
   port: parseInt(process.env.UNMS_RABBITMQ_PORT, 10) || 5672,
+  user: process.env.UNMS_RABBITMQ_USER || '',
+  pass: process.env.UNMS_RABBITMQ_PASS || '',
   exchange: 'unms',
   queues: {
     settingsChanged: 'settings.changed',